frontend/package.json

```diff
@@ -51,4 +51,4 @@
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-react": "^7.31.11"
   }
-}
+}
```

Comment:
please add empty line at the end

Reply:
Hi @furkanmutlu  I tried to add empty line at the end but my IDE is ignoring that. The above mentioned change is due to code formatter i think and it is looking fine to me. But if you still want me to change that then i will change my IDE settings and do that?

Reply:
for VS Code, for example, there is this setting you can setup:
![image](https://github.com/MutluMinds/fullstack-image-app/assets/45897337/3aa81ab2-1b63-4191-9d18-90a01ecef906)

Reply:
Let me know if that works for you if not I can fix it

Reply:
Hi @furkanmutlu Thanks for your help. I have added new lines at proper places and created a PR to merge those changes. Please review it.